frontend/src/metabase/components/ModalContent/ModalContent.tsx

```diff
@@ -82,6 +82,9 @@ export default class ModalContent extends Component<ModalContentProps> {
           // add bottom padding if this is a standard "form modal" with no footer
           { pb4: formModal && !footer },
         )}
+        role="dialog"
+        aria-label={title}
+        aria-modal="true"
```

Comment:
Tests that look for a specific modal look for the `ModalContent` component, which has these attributes so it can be found semantically in a unit test.

Why not put these attributes on one of the ancestors of the `ModalContent` component? The reason is that these ancestors don't receive a `title` prop.